README.md

```diff
@@ -3,60 +3,79 @@
 </p>
 
 <p align="center">
-<i>Collection of Dockerfile and docker-compose configurations wrapped in an easy-to-use command line, oriented for PHP projects.</i>
+    <i>Collection of Dockerfile and docker-compose configurations wrapped in an easy-to-use command line, oriented for PHP projects.</i>
 </p>
 
 ## What is Docker Starter Kit
 
-This repository provides a Docker infrastructure for your PHP projects with built-in support for HTTPS, custom domain, databases, workers...
-and is used as a foundation for our projects here at [JoliCode](https://jolicode.com/).
+This repository provides a Docker infrastructure for your PHP projects with
+built-in support for HTTPS, custom domain, databases, workers... and is used as
+a foundation for our projects at [JoliCode](https://jolicode.com/).
 
 Learn more about it and why we created and open-sourced this starter-kit in our blog posts:
 [in English 🇬🇧](https://jolicode.com/blog/introducing-our-docker-starter-kit) or
 [in French 🇫🇷](https://jolicode.com/blog/presentation-de-notre-starter-kit-docker).
 
 > **Warning**
-> You are reading the README of version 3 that uses [Invoke](https://www.pyinvoke.org/).
+> You are reading the README of version 4 that uses [castor](https://github.com/jolicode/castor).
 
+* If you are using [Invoke](https://www.pyinvoke.org/), you can read the [dedicated README](https://github.com/jolicode/docker-starter/tree/v3.0.0);
+* If you want to migrate from docker-starter v2.x to v3.0, you can read the [dedicated guide](./UPGRADE-4.0.md);
```

Comment:
v2 to v3 but you linked the UPGRADE-4.0.md, typo?

Reply:
Let's drop the guide. Too complicated to migrate! a full rewrite is needed.